src/app/scripts/cac/control/cac-control-sidebar-explore.js

```diff
@@ -203,6 +204,47 @@ CAC.Control.SidebarExplore = (function ($, BikeOptions, MapTemplates, Typeahead,
         $('div.address > h4').html(MapTemplates.addressText(location.feature.attributes));
     }
 
+    /**
+     * Query OTP for travel time to a destination, then put it in the side panel.
+     *
+     * @param {Object} destination Destination put in the sidebar
+     * @param {Object} $container jQuery-selected HTML snippet in the sidebar for the destination
+     */
+    function setDestinationDistance(destination, $container) {
+        var mode = $(options.selectors.modeSelector).val();
+        var picker = $(options.selectors.datepicker).data('DateTimePicker');
+        var date = picker.date();
+        if (!date) {
+            // use current date/time if none set
+            date = moment();
+        }
+        // only request one itinerary (first returned is the shortest)
+        var otpOptions = { mode: mode, numItineraries: 1 };
+        if (mode.indexOf('BICYCLE') > -1) {
+            var bikeTriangleOpt = $('option:selected', options.selectors.bikeTriangleDiv);
```

Comment:
Really probably going to want to refactor getting these values somehow if they get pulled in any more places.